http/src/main/resources/reference.conf

```diff
@@ -390,7 +390,7 @@ azure {
    }
 
    hail-batch-app-config {
-     chart-name = "/leonardo/hail-batch-terra-azure"
+     chart-name = "oci://terradevacrpublic.azurecr.io/hail/hail-batch-terra-azure"
```

Comment:
Looks like OCI is supported as of helm 3.8.0: https://helm.sh/docs/topics/registries/

I was wondering if we need to update helm but seems we're on 3.11.2 today: https://github.com/DataBiosphere/leonardo/blob/develop/Dockerfile#L52

Reply:
We might very well be good, I have not yet tested hail batch on my BEE yet :)

Reply:
Hm -- 👁️

Reply:
The more I read on the subject, the more I suspect that we need to pull the chart before installing. I don't think that install alone is supported for OCI